distribution/openrct2.d.ts

```diff
@@ -2076,6 +2076,61 @@ declare global {
          * The min chain lift speed for this ride in miles per hour.
          */
         readonly minLiftHillSpeed: number;
+
+        /**
+         * The max speed in miles per hour.
+         */
+        readonly maxSpeed: number;
+
+        /**
+         * The average speed in miles per hour.
+         */
+        readonly averageSpeed: number;
+
+        /**
+         * The ride time in seconds.
+         */
+        readonly rideTime: number;
+
+        /**
+         * Total length of the ride in meters.
+         */
+        readonly rideLength: number;
+
+        /**
+         * The max positive vertical Gs in hundredths of a G.
+         */
+        readonly maxPositiveVerticalGs: number;
+
+        /**
+         * The max negative vertical Gs in hundredths of a G.
+         */
+        readonly maxNegativeVerticalGs: number;
+
+        /**
+         * The max lateral Gs in hundredths of a G.
+         */
+        readonly maxLateralGs: number;
+
+        /**
+         * The total airtime in hundredths of a second.
+         */
+        readonly totalAirTime: number;
+
+        /**
+         * The number of drops.
+         */
+        readonly drops: number;
+
+        /**
+         * The number of powered lifts.
+         */
+        readonly poweredLifts: number;
+
+        /**
+         * Highest drop height in height units. Use `formatString()` to convert into metres/feet. Ex: `formatString('{HEIGHT}', ride.highestDropHeight)`.
```

Comment:
Why explain it here but nowhere else? E.g. for the speed.

Reply:
I feel that formatString() isn't that well documented anywhere, so I added my own comments for the one I implemented. Sure, I can add it for the rest of the Ride interface, but it probably should've been there to begin with.